### DISCUSSION TITLE
cant install nvim-neorocks under windows

### DISCUSSION THREAD

**Discussion by MarvinHauke** [lumen-oss/rocks.nvim#358] [Q&A] [answered]
I always get this error while trying to install with the install script under windows.
can some one help me?
are there any additioal informations I need to provide?

![image](https://github.com/nvim-neorocks/rocks.nvim/assets/50217945/d98457e1-87c6-4077-b7a1-460ab47dfb84)

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

The error is thrown here, because you don't have `sh` or git bash installed:

https://github.com/nvim-neorocks/rocks.nvim/blob/ccdaaa1cb1f5484575c81f452f1961ee1a005d51/installer.lua?plain=1#L192-L194

We definitely need to wrap those calls to output nicer error messages:

- https://github.com/nvim-neorocks/rocks.nvim/issues/359
